docs/src/features/atlases.md

```diff
@@ -1,21 +1,34 @@
 # [Atlases and charts](@id atlases_and_charts)
 
-Atlases on an $n$-dimensional manifold $\mathcal M$ are collections of charts $\{(U_i, \varphi_i) \colon i \in I\}$ such that $U_i \subseteq \mathcal M$ and each chart $\varphi_i$ is a function from $U_i$ to $\mathbb{R}^n$.
-They provide a basic connection between manifolds and Euclidean spaces.
+Atlases on an ``n``-dimensional manifold $\mathcal M$ are collections of charts ``\mathcal A = \{(U_i, \varphi_i) \colon i \in I\}``, where ``I`` is a (finite or infinte) index family, such that ``U_i \subseteq \mathcal M`` is an open set and each chart ``\varphi_i: U_i \to \mathbb{R}^n`` is a homeomorphism. This means, that ``\varphi_i`` is bijecive – sometimes also called one-to-one and onto - and continuous, and its inverse ``\varphi_i^{-1}`` is continuous as well.
+The inverse ``\varphi_i^{-1}`` is called (local) parametrization.
+For an atlas ``\mathcal A`` we further require that
+
+```math
+\displaystyle\bigcup_{i\in I} U_i = \mathcal M.
+```
+
+We say ``\varphi_i`` is a chart about ``p``, if ``p\in U_i``
+An atlas provides a connection between a manifold and the Euclidean space ``\mathbb{R}^n``, since
+locally, a chart about ``p`` can be used to identify its neighborhood (as long as you sta in ``U_i``) “looks like” the Euclidean space.
```

Comment:
```suggestion
locally, a chart about ``p`` can be used to identify its neighborhood (as long as you stay in ``U_i``) with a subset of a Euclidean space.
```
Feel free to rephrase it, I just didn't like the original version.

Reply:
Your suggestion is fine, my text was too informal, I agree.